contracts/contracts/Umbra.sol

```diff
@@ -120,6 +120,50 @@ contract Umbra is BaseRelayRecipient, IKnowForwarderAddress, Ownable {
     SafeERC20.safeTransfer(IERC20(tokenAddr), _acceptor, amount);
   }
 
+  function withdrawMeta(
+    address _stealthAddr,
+    address _sponsor,
```

Comment:
I actually intentionally chose `sponsor` rather than `relayer` for this parameter, because I think there is a subtle difference here. For example, I'd consider the account submitting the TX to the chain the "relayer", but that might not be the account receiving the payout (for example, the relayer service may choose to send payouts to an non-hot address).

Reply:
Fair enough!